clientv3/integration/watch_test.go

```diff
@@ -343,16 +343,45 @@ func putAndWatch(t *testing.T, wctx *watchctx, key, val string) {
 	}
 }
 
-// TestWatchResumeComapcted checks that the watcher gracefully closes in case
+func TestWatchResumeCompacted(t *testing.T) {
+	defer testutil.AfterTest(t)
+	testWatchResumeCompacted(t, nil, nil)
+}
+
+func TestWatchResumeCompactedFrequentDisconnect(t *testing.T) {
```

Comment:
I can try some ways to do deterministic without failpoints but it'll change the behavior that's being tested. OK, whatever.